test/ng/testabilitySpec.js

```diff
@@ -189,10 +189,35 @@ describe('$$testability', function() {
 
   describe('waiting for stability', function() {
     it('should process callbacks immediately with no outstanding requests',
-      inject(function($$testability) {
+      inject(function($$testability, $timeout) {
```

Comment:
The description of this test looks incorrect now (as nothing gets invoked immediately).